packages/runtime/runtime-utils/src/index.ts

```diff
@@ -9,6 +9,7 @@ export * from "./objectstoragepartition";
 export * from "./objectstorageutils";
 export * from "./requestParser";
 export * from "./serializer";
-export * from "./summarizerNode";
+// eslint-disable-next-line import/no-internal-modules
+export * from "./summarizerNode/index";
```

Comment:
Curious, why do you need to make this change? I think it's equivalent to prior code, no?

Reply:
idk how it got there hmmm